pkg/cmd/adm/restart_test.go

```diff
@@ -1,216 +1,390 @@
+			} else if tc.name1 == "autoscaling-buffer" {
+				require.Contains(t, term.Output(), "Found only autoscaling-buffer deployment in namespace toolchain-member-operator , which is not required to be restarted")
+				assert.Equal(t, 1, csCalls)
+			}
 
-			// then
-			require.NoError(t, err)
-			AssertDeploymentHasReplicas(t, fakeClient, namespacedName, 3)
-			assert.Equal(t, 0, numberOfUpdateCalls)
-			assert.Contains(t, term.Output(), "ERROR: The given deployment 'wrong-deployment' wasn't found.")
-			assert.Contains(t, term.Output(), fmt.Sprintf("Existing deployments in toolchain-%s-operator namespace", clusterType))
-			assert.Contains(t, term.Output(), "cool-deployment")
 		})
 	}
 }
 
-func TestRestartDeploymentWithInsufficientPermissions(t *testing.T) {
-	// given
-	SetFileConfig(t, Host(NoToken()), Member(NoToken()))
-	for _, clusterName := range []string{"host", "member1"} {
-		// given
-		clusterType := configuration.Host
-		if clusterName != "host" {
-			clusterType = configuration.Member
-		}
-		namespace := fmt.Sprintf("toolchain-%s-operator", clusterType)
-		namespacedName := types.NamespacedName{
-			Namespace: namespace,
-			Name:      "cool-deployment",
-		}
-		deployment := newDeployment(namespacedName, 3)
-		newClient, fakeClient := NewFakeClients(t, deployment)
-		numberOfUpdateCalls := 0
-		fakeClient.MockUpdate = requireDeploymentBeingUpdated(t, fakeClient, namespacedName, 3, &numberOfUpdateCalls)
-		term := NewFakeTerminalWithResponse("Y")
+func TestRestartDeployment(t *testing.T) {
+	//given
+	testIOStreams := genericclioptions.NewTestIOStreamsDiscard()
+	SetFileConfig(t, Host())
+	toolchainCluster := NewToolchainCluster(ToolchainClusterName("host"))
+	hostDeployment := newDeployment(test.NamespacedName("toolchain-host-operator", "host-operator-controller-manager"), 1)
+	hostDeployment.Labels = make(map[string]string)
+	hostDeployment.Labels["kubesaw-control-plane"] = "kubesaw-controller-manager"
+	regServDeployment := newDeployment(test.NamespacedName("toolchain-host-operator", "registration-service"), 1)
+	regServDeployment.Labels = make(map[string]string)
+	regServDeployment.Labels["toolchain.dev.openshift.com/provider"] = "codeready-toolchain"
+	hostPod := newPod(test.NamespacedName("toolchain-host-operator", "host-operator-controller-manager"))
+	noisePod := newPod(test.NamespacedName("toolchain-host-operator", "noise"))
+	memberDeployment := newDeployment(test.NamespacedName("toolchain-member-operator", "member-operator-controller-manager"), 1)
+	memberDeployment.Labels = make(map[string]string)
+	memberDeployment.Labels["kubesaw-control-plane"] = "kubesaw-controller-manager"
+	autoscalarDeployment := newDeployment(test.NamespacedName("toolchain-member-operator", "autoscaling-buffer"), 1)
+	autoscalarDeployment.Labels = make(map[string]string)
+	autoscalarDeployment.Labels["toolchain.dev.openshift.com/provider"] = "codeready-toolchain"
+	actualPod := &corev1.Pod{}
+	term := NewFakeTerminalWithResponse("Y")
```

Comment:
this is pretty hard to read, could you please separate the related lines by an empty line and by comments? something like this
```go
	testIOStreams := genericclioptions.NewTestIOStreamsDiscard()
	SetFileConfig(t, Host())
	toolchainCluster := NewToolchainCluster(ToolchainClusterName("host"))
	term := NewFakeTerminalWithResponse("Y")

	// OLM-based deployments
	hostDeployment := newDeployment(test.NamespacedName("toolchain-host-operator", "host-operator-controller-manager"), 1)
	hostDeployment.Labels = map[string]string{}
	hostDeployment.Labels["kubesaw-control-plane"] = "kubesaw-controller-manager"
	hostPod := newPod(test.NamespacedName("toolchain-host-operator", "host-operator-controller-manager"))

	memberDeployment := newDeployment(test.NamespacedName("toolchain-member-operator", "member-operator-controller-manager"), 1)
	memberDeployment.Labels = make(map[string]string)
	memberDeployment.Labels["kubesaw-control-plane"] = "kubesaw-controller-manager"

	// non-OLM-based deployments
	regServDeployment := newDeployment(test.NamespacedName("toolchain-host-operator", "registration-service"), 1)
	regServDeployment.Labels = make(map[string]string)
	regServDeployment.Labels["toolchain.dev.openshift.com/provider"] = "codeready-toolchain"

	autoscalarDeployment := newDeployment(test.NamespacedName("toolchain-member-operator", "autoscaling-buffer"), 1)
	autoscalarDeployment.Labels = make(map[string]string)
	autoscalarDeployment.Labels["toolchain.dev.openshift.com/provider"] = "codeready-toolchain"

	actualPod := &corev1.Pod{}
	noisePod := newPod(test.NamespacedName("toolchain-host-operator", "noise"))
```

Reply:
changed